tests/rules/javascript/test.content_scripts_file_absent.js

```diff
@@ -0,0 +1,191 @@
+import { VALIDATION_ERROR } from 'const';
+import { CONTENT_SCRIPT_NOT_FOUND, CONTENT_SCRIPT_EMPTY } from 'messages/javascript';
+import JavaScriptScanner from 'scanners/javascript';
+
+describe('content_scripts_file_absent', () => {
+  it('should show an error when content script is missing', () => {
+    const code = `browser.tabs.executeScript({file: '/content_scripts/absentFile.js'});`;
+    const fileRequiresContentScript = 'file-requires-content-script.js';
+    const jsScanner = new JavaScriptScanner(code, fileRequiresContentScript, {
+      existingFiles: {
+        fileRequiresContentScript,
+      },
+      addonMetadata: {
+        id: 'test',
+      },
+    });
+
+    return jsScanner.scan()
+      .then(({ linterMessages }) => {
+        linterMessages.sort();
+        expect(linterMessages.length).toEqual(1);
+        expect(linterMessages[0].code).toEqual(CONTENT_SCRIPT_NOT_FOUND.code);
+        expect(linterMessages[0].type).toEqual(VALIDATION_ERROR);
+      });
+  });
+
+  it('should not show an error when content script is not missing', () => {
+    const code = `browser.tabs.executeScript({file: '/content_scripts/existingFile.js'});`;
+    const fileRequiresContentScript = 'file-requires-content-script.js';
+    const jsScanner = new JavaScriptScanner(code, fileRequiresContentScript, {
+      existingFiles: {
+        fileRequiresContentScript,
+        'content_scripts/existingFile.js': '',
+      },
+      addonMetadata: {
+        id: 'test',
+      },
+    });
+
+    return jsScanner.scan()
+      .then(({ linterMessages }) => {
+        expect(linterMessages.length).toEqual(0);
```

Comment:
On the assertion that checks that the validation messages are empty, it could be better to assert that the 
`linterMessages` value is equal to an empty array, so that when the assertion fails it prints the unexpected validation errors in the failure message, e.g. `expect(linterMessages).toEqual([]);` would raise a failure message like this:

```
 expect(received).toEqual(expected)
    
    Expected value to equal:
      []
    Received:
      [{"code": "CONTENT_SCRIPT_EMPTY", "column": 35, "description": "Content script file name should not be empty", "file": "file-with-code.js", "line": 1, "message": "Content script file name should not be empty.", "sourceCode": "browser.tabs.executeScript({file: '' });", "type": "error"}]
    
    Difference:
    
    - Expected
    + Received
    
    - Array []
    + Array [
    +   Object {
    +     "code": "CONTENT_SCRIPT_EMPTY",
    +     "column": 35,
    +     "description": "Content script file name should not be empty",
    +     "file": "file-with-code.js",
    +     "line": 1,
    +     "message": "Content script file name should not be empty.",
    +     "sourceCode": "... /* the validated source code */ ...",
    +     "type": "error",
    +   },
    + ]
```

And the above error would make it way easier to guess the reasons behind a failure.